app/autodiscovery/src/main/java/com/fsck/k9/autodiscovery/providersxml/ProvidersXmlDiscovery.kt

```diff
@@ -17,6 +17,10 @@ class ProvidersXmlDiscovery(
 ) : ConnectionSettingsDiscovery {
 
     override fun discover(email: String): ConnectionSettings? {
+        return discover(email, outgoing = true, incoming = true)
```

Comment:
I suggest replacing the parameters `incoming` and `outgoing` with an enum class, so that the invalid state where both arguments are `false` will be prevented by the type system.